topics/centering.rst

```diff
@@ -14,7 +14,7 @@ the vertical axis with the current :math:`L_z` axis, which we call
 :math:`J`-centering. Both may be used together.
 
 The algorithms for both centering types rely on the same logic as
-follows.  The user specifies the desired number of states for
+follows.  The user specifies the desired number of particles for
```

Comment:
Perhaps EJwant and EJkeep could be changed to make them more descriptive monikers?  Not the most inspired choices perhaps.